fireant/dataset/filters.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, field, start, stop):
 
     @property
     def definition(self):
-        return self.field.definition[self.start : self.stop]
+        return self.field.definition[self.start: self.stop]
```

Comment:
Minor: Seems like strange formatting. Not sure Black is setup on Fireant yet.

Reply:
My IDE did this. I guess no space either after the colon would make more sense.